Dockerfile

```diff
@@ -70,6 +70,8 @@ ARG TARGETPLATFORM
 RUN echo "Target platform is $TARGETPLATFORM"
 
 RUN \
+# Remove automatic apt cache Docker cleanup scripts
```

Comment:
it's fine to keep here - my preference however is to have it just before the "apt" commands in line ~92, so they are located near each other, so if it gets refactored for whatever reason, it will automatically "move" over since its the same `RUN` block.

Similarly, I also like to repeat the `rm` command in every `RUN` that touches `apt`, even if its already gone, to avoid any future refactoring moving things around and then suddenly having a regression of non-utilized apt caches